src/EditorFeatures/CSharpTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -1855,11 +1855,16 @@ class A { }");
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsSimplifyTypeNames)]
         public void TestMissingNullableSimplificationInsideCref4()
         {
-            TestMissing(
+            Test(
 @"using System;
```

Comment:
Test bug.  We do offer to simplify here.  However, TestMissing never did anything, so we missed this.